web/src/styles/notebook.css

```
@@ -114,99 +130,159 @@
+  padding: 0 4px 16px;
 }
 .notebook-surface .nb-shelf-head h2 {
   font-family: var(--nb-display);
   font-size: 22px;
   font-weight: 600;
   color: var(--nb-text);
   line-height: 1.1;
+  margin: 0;
 }
 .notebook-surface .nb-shelf-head .nb-shelf-role {
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)